easybuild/easyconfigs/h/hivtrace/hivtrace-0.6.2-foss-2021a.eb

```diff
@@ -0,0 +1,72 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Iñaki Mtz de Ilarduya Muñoz
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'PythonBundle'
+
+name = 'hivtrace'
+version = '0.6.2'
+
+homepage = 'https://github.com/veg/hivtrace'
+description = """HIV-TRACE is an application that identifies potential transmission clusters within a
+ supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('Biopython', '1.79'),
+    ('tn93', '1.0.7'),
+    ('cURL', '7.76.0'),
+    ('Pysam', '0.16.0.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('hyphy-python', '0.1.10', {
+        'modulename': 'HyPhy',
+        'source_urls': ['https://pypi.python.org/packages/source/h/hyphy-python'],
```

Comment:
@i-mtz the source_urls can be removed for all the extensions, as pypi is the default for extensions.